### PR TITLE
5.2 cleanup theme_barceloneta WIP

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

File: bobtemplates/plone/theme_barceloneta/theme/barceloneta/BARCELONETA-VERSION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+2.1.3
```
Comment (MrTango) [inline bobtemplates/plone/theme_barceloneta/theme/barceloneta/BARCELONETA-VERSION.rst]: i would move this file one level up

**Comment by djowett** [comment]
Does this apply to 5.1 as well as 5.2?

**Comment by agitator** [comment]
@djowett barceloneta 2.1.2 should work with Plone 5.1 and 5.2, so these changes should also work for both version.
Not sure though why http://dist.plone.org/release/5.1.6-pending/versions.cfg wasn't updated with  https://github.com/plone/buildout.coredev/blob/5.1/versions.cfg#L290 yet

**Comment by djowett** [comment]
Thanks @agitator

**Comment by balavec** [comment]
There are also some file permission changes from 100644 → 100755. Is this intentional?

**Comment by agitator** [comment]
Todo:
- fix robot font path
- update Readme

**Pull request merged**